social_recovery/contracts/libraries/CreateAndAddModules.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity >=0.5.0 <0.7.0;
+import "@gnosis.pm/safe-contracts/contracts/base/Module.sol";
+
+
+/// @title Create and Add Modules - Allows to create and add multiple module in one transaction.
+/// @author Stefan George - <stefan@gnosis.pm>
+/// @author Richard Meissner - <richard@gnosis.pm>
+contract CreateAndAddModules {
```

Comment:
Why do we need this here? This contract should be part of the gnosis safe package